tests/testthat/test_single_var_association_tests.R

```diff
@@ -201,9 +201,22 @@ test_that("small null model", {
     test.cmp.small2 <- testGenoSingleVar(nullmod.small2, G = geno, test = "CMP")
     expect_equal(test.cmp.big, test.cmp.small2)
 
-    expect_error(testGenoSingleVar(nullmod.small2, G = geno, test = "BinomiRare",
-                                   recalc.pval.thresh=0.5),
-                 "small null model cannot be used")
-    expect_error(testGenoSingleVar(nullmod.small2, G = geno, test = "Score"),
-                 "small null model cannot be used")
+    # expect_error(testGenoSingleVar(nullmod.small2, G = geno, test = "BinomiRare",
+    #                                recalc.pval.thresh=0.5),
+    #              "small null.model cannot be used with test options provided")
+    # expect_error(testGenoSingleVar(nullmod.small2, G = geno, test = "Score"),
+    #              "small null.model cannot be used with test options provided")
+
```

Comment:
@amstilp can you please take a look at these. the checks below replaced the ones commented out above. we moved where those checks occur in the code. i'm not sure if these new checks should be a new "section" in the code, or just included where they are.

Reply:
I would change these to running `testGenoSingleVar` instead of `.checkNullModelTestSingle` to make sure that you're getting the correct error from `testGenoSingleVar` with the options provided, so for example test this:
```
    expect_error(testGenoSingleVar(nullmod.small2, G = geno, test = "BinomiRare",
        recalc.pval.thresh = 0.5, fast.score.SE = FALSE, GxE = NULL),
        "small null.model cannot be used with test options provided")
```

instead of this:

```
    expect_error(.checkNullModelTestSingle(nullmod.small2, test = "BinomiRare",
        recalc.pval.thresh = 0.5, fast.score.SE = FALSE, GxE = NULL),
        "small null.model cannot be used with test options provided")
```

That way it's harder to accidentally remove the step that checks the null model from the `testGenoSingleVar` function.

Reply:
Oh wait, I see. The checks happen in `assocTestSingle` instead. Ignore my above comment.

Reply:
In that case, I would make a new test specifically for testing `.checkNullModelTestSingle`. You could also add a check (or tests?) where it succeeds without an error. I think you can use expect_silent for that, or check that it returned an invisible NULL?

Reply:
Not a must have, since the regular single variant association tests aren't breaking. I'm a big fan of unit tests but I think it's fine to write them later given that check and the deadline. So let's leave this test as is for now and then write the more complete test later?